boards/qorvo/decawave_dwm3001cdk/Kconfig.defconfig

```diff
@@ -0,0 +1,60 @@
+# DecaWave DWM3001CDK board configuration
+
+# Copyright (c) 2019 Stéphane D'Alu
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_DECAWAVE_DWM3001CDK
+
+config BT_CTLR
+	default BT
+
+config I2C
+	default SENSOR
+
+if BOARD_SERIAL_BACKEND_CDC_ACM
```

Comment:
I took this from boards/st/sensortile_box_pro cause it made the `tests/subsys/llext/simple` test build, I don't really know if it's correct